packages/mobile-app/data/facades/wallet/oreowalletHandlers.ts

```diff
@@ -0,0 +1,385 @@
+import { f } from "data-facade";
+import {
+  Account,
+  AccountSettings,
+  Output,
+  Transaction,
+  TransactionType,
+  WalletHandlers,
+  WalletStatus,
+} from "./types";
+import { oreoWallet } from "../../wallet/oreowalletWallet";
+import { Network } from "../../constants";
+import * as Uint8ArrayUtils from "../../../utils/uint8Array";
+
+import {
+  AccountFormat,
+  LanguageKey,
+  LanguageUtils,
+  TransactionStatus,
+} from "@ironfish/sdk";
+import { OreowalletServerApi } from "../../oreowalletServerApi/oreowalletServerApi";
+
+const IRON_ASSET_ID_HEX =
+  "51f33a2f14f92735e562dc658a5639279ddca3d5079a6d1242b2a588a9cbf44c";
```

Comment:
suggestion: Might make sense to put this in a shared constants file somewhere. I'm guessing we'll need it in other places.

Reply:
Done